charts/rancher-gke-operator/1.0.100/questions.yaml

```
@@ -0,0 +1 @@
+rancher_min_version: 2.5.7-alpha1
```

@cmurphy Shouldn't this be: 
```suggestion
rancher_min_version: 2.5.8-alpha1 ? 
```

---

@cbron yes it should

---

https://github.com/rancher/system-charts/pull/455